README.md

```diff
@@ -31,6 +31,20 @@ To build the binaries with debug symbols, run:
 make debug
 ```
 
+Note: build process requires pkg-config to be installed:
+
+### Mac OS X
+
+```
+curl https://pkgconfig.freedesktop.org/releases/pkg-config-0.29.tar.gz -o pkgconfig.tgz
```

Comment:
These version numbers will not last.

Reply:
removed specific version of pkg-config. User can specify version that fits him the best.